perception/autoware_elevation_map_loader/launch/elevation_map_loader.launch.xml

```diff
@@ -13,11 +13,6 @@
     <remap from="input/pointcloud_map_metadata" to="/map/pointcloud_map_metadata"/>
     <remap from="input/vector_map" to="/map/vector_map"/>
     <remap from="service/get_selected_pcd_map" to="/map/get_selected_pointcloud_map"/>
-
-    <param name="elevation_map_directory" value="$(var elevation_map_directory)"/>
```

Comment:
@oguzkaganozt 

Why do you leave you remove `elevation_map_directory` arg although you removed `elevation_map_directory` param ?